debugger/Makefile

```diff
@@ -73,7 +72,7 @@ all_objects := $(addsuffix .cmo,$(all_modules))
 
 all: ocamldebug$(EXE)
 
-ocamldebug$(EXE): $(UNIXDIR)/unix.cma $(all_objects)
+ocamldebug$(EXE): $(UNIXDIR)/unix.cma $(DYNLINKDIR)/dynlink.cma $(all_objects)
```

Comment:
Not sure this is right; this means that `ocamldebug` will contain "two" copies of compiler-libs (cf the line that says "REMOVE ME for ..." in `otherlibs/dynlink/dynlink.ml`.

Reply:
We can probably ditch the "REMOVE ME" stuff entirely (this could be done in #2268).  I don't think there is any problem with the debugger containing the (relatively small) subset of compilerlibs in the dynlink-specific pack, in addition to any other compilerlibs modules it may or may not link in.  Removing the special case has more value, I think.

Reply:
I may be mistaken, but my understanding was that `Dynlink` may modify some state of `compiler-libs` that is then accessed by the debugger. This kind of code would break if we do this.

Reply:
@damiendoligez Could you please comment?

Reply:
I have only a vague idea of what you guys are talking about. @xclerc should be more competent.

Reply:
I was thinking about the following code, which is used to load user-defined printers into the debugger. 

https://github.com/ocaml/ocaml/blob/f785be31bdeb8741bb2bc64554cfdf72e3c2c45d/debugger/loadprinter.ml#L33-L69

Reply:
I'm probably going to have to sit and think about this more carefully, but my initial impression here is that this code could maybe be removed as well.  With this pull request, the copy of `Symtable` that `Dynlink` uses is different from the one linked directly into the debugger.

Reply:
@xclerc and I are looking at the debugger code; this patch isn't quite right in that area yet, it would seem.

Reply:
The debugger problem is now fixed.